scripts/system/html/gridControls.html

```diff
@@ -97,9 +88,9 @@ <h2>Voxel edit settings</h2>
             <div class="tools-help-popup" id="voxel-help-popup" >
                 <p>To edit voxels, Voxel Edit Mode needs to be selected.</p>
                 <p>Desktop mode:</p>
-                <p>Click left mouse button to add voxel. Click middle mouse button to remove voxel. Hold mouse button and move mouse to add/remove voxels in a single plane. Plane is determined by the direction where you are looking when first voxel is added/removed (for example look downwards for horizontal plane.</p>
+                <p>Click the left mouse button to add voxel. Click the middle mouse button to remove voxel. Hold the mouse button and move mouse to add/remove voxels in a single plane. Plane is determined by the direction in which you are looking when first voxel is added/removed (for example look downwards to draw in horizontal plane).</p>
```

Comment:
I'd change this a bit more:

```suggestion
                <p>Click the left mouse button to add a voxel. Click the middle mouse button to remove a voxel. Hold the mouse button and move the mouse to add or remove voxels in a single plane. The plane is determined by the direction in which you are looking when first voxel is added or removed (for example, look downwards to draw in horizontal plane).</p>
```